elevation_service/BUILD.gn

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+
+assert(is_win)
+
+source_set("elevation_service") {
+  sources = [
+    "install_utils.cc",
+    "install_utils.h",
+  ]
+
+  public_deps = [ "//brave/components/brave_vpn/common/wireguard/win:brave_wireguard_manager_idl" ]
+  deps = [
+    "//base",
+    "//brave/browser/brave_vpn/win/brave_vpn_wireguard_service/service",
+    "//brave/browser/brave_vpn/win/brave_vpn_wireguard_service/status_tray",
+    "//brave/components/brave_vpn/common/win",
+    "//brave/components/brave_vpn/common/wireguard/win",
+    "//chrome/installer/util:with_no_strings",
+  ]
+  libs = [
+    "Shell32.lib",
+
+    #NOTE: need to clean this up
```

Comment:
please change this to TODO and link to an issue

Reply:
done - created https://github.com/brave/brave-browser/issues/35288